src/java/org/jivesoftware/openfire/index/LuceneIndexer.java

```diff
@@ -409,6 +439,11 @@ public Future<Integer> getIndexRebuildProgress()
      */
     public synchronized IndexSearcher getSearcher() throws IOException
     {
+        if (!ENABLED.getValue()) {
+            Log.debug("Unable to get index: indexing is disable by configuration.");
```

Comment:
```suggestion
            Log.debug("Unable to get index: indexing is disabled by configuration.");
```

Reply:
Ah, I replaced 4 of these in a commit that I just pushed.